acm/src/main/java/org/literacybridge/acm/core/DataRequestResult.java

```diff
@@ -4,70 +4,52 @@
 import java.util.Map;
 
 import org.literacybridge.acm.api.IDataRequestResult;
-import org.literacybridge.acm.categories.Taxonomy.Category;
-import org.literacybridge.acm.db.PersistentTag;
+import org.literacybridge.acm.store.Category;
 
 public class DataRequestResult implements IDataRequestResult {
```

Comment:
As the concrete implementation of IDataRequestResult, this should probably document just what it (concretely) is.